lib/git-fastclone.rb

```diff
@@ -162,14 +162,15 @@ def parse_options
           options[:config] = config
         end
 
-        opts.on('--lock-timeout N', 'Timeout in seconds to acquire a lock on any reference repo.
-                Default is 0 which waits indefinitely.') do |timeout_secs|
+        opts.on('--lock-timeout N', 'Timeout in seconds to acquire a lock on any reference repo.',
+                'Default is 0 which waits indefinitely.') do |timeout_secs|
```

Comment:
Small cleanup to have right \t
```
Usage: git fastclone [options] <git-url> [path]
    -b, --branch BRANCH              Checkout this branch rather than the default
    -v, --verbose                    Verbose mode
        --print_git_errors           Print git output if a command fails
    -c, --color                      Display colored output
        --config CONFIG              Git config applied to the cloned repo
        --lock-timeout N             Timeout in seconds to acquire a lock on any reference repo.
                                     Default is 0 which waits indefinitely.
        --pre-clone-hook script_file An optional file that should be invoked before cloning mirror repo
                                     No-op when a file is missing
```